src/material/radio/radio.ts

```diff
@@ -585,24 +585,23 @@ export abstract class _MatRadioButtonBase extends _MatRadioButtonMixinBase imple
     event.stopPropagation();
   }
 
-  /**
-   * Triggered when the radio button received a click or the input recognized any change.
-   * Clicking on a label element, will trigger a change event on the associated input.
-   */
-  _onInputChange(event: Event) {
+  /** Triggered when the radio button receives an interaction from the user. */
+   _onInputInteraction(event: Event) {
     // We always have to stop propagation on the change event.
     // Otherwise the change event, from the input element, will bubble up and
     // emit its event object to the `change` output.
     event.stopPropagation();
```

Comment:
Should we check the event type here so we don't stop propagation on the touch target click events? Or are we intentionally stopping those too?

Reply:
We were stopping all events on the radio so I assumed that we wanted to do the same for the touch target.